src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc($cell-size * 7 + $gap * 6);
```

Comment:
Ensure that the SCSS variables `$cell-size` and `$gap` are defined in your SCSS files. Without these definitions, the `calc` function will not work correctly.